src/mesh/installation/ecs.md

```diff
@@ -46,56 +44,39 @@ This section covers ECS-specific parts of running Kong Mesh, using the
 
 Kong Mesh runs in Universal mode on ECS. The example setup repository uses an AWS RDS
 database as a PostgreSQL backend. It also uses ECS service discovery to enable ECS
-tasks to communicate with Kong Mesh the control plane.
+tasks to communicate with the Kong Mesh control plane.
 
 The example Cloudformation includes two Cloudformation stacks for
-[creating a cluster](https://github.com/Kong/kong-mesh-ecs/blob/main/deploy/vpc.yaml) and
-[deploying Kong Mesh](https://github.com/Kong/kong-mesh-ecs/blob/main/deploy/controlplane.yaml)
+[creating a cluster](https://github.com/Kong/kong-mesh-ecs/blob/workload-identity/deploy/vpc.yaml) and
+[deploying Kong Mesh](https://github.com/Kong/kong-mesh-ecs/blob/workload-identity/deploy/controlplane.yaml)
 
-### Kong Mesh ECS controller
+#### Workload identity
```

Comment:
As all this is new should it be guarded with `{% if_version gte:2.0.x %}` ? @cloudjumpercat to help :)

Reply:
Yep, anything that is specific to 2.0 will need `{% if_version gte:2.0.x %}` and `{% endif_version %}` around it. More info here: https://github.com/Kong/docs.konghq.com/blob/main/docs/single-sourced-versions.md#conditional-rendering

Reply:
And anything that was removed for 2.0 but exists in previous versions should have `{% if_version lte:1.9.x %}` and `{% endif_version %}` around it instead of removing it completely from the doc

Reply:
I've just created two entire pages for each version.